CONTRIBUTING.md

```diff
@@ -126,6 +126,9 @@ From there you will have to wait on one of the GAP committers to respond to
 the request. This response might be an accept or some
 changes/improvements/alternatives will be suggested.  We do not guarantee
 that all requests will be accepted.
+Feel free to read the section
+[the reviewing process](https://github.com/gap-system/gap/blob/master/CONTRIBUTING.md#the-reviewing-process)
```

Comment:
This hardcodes a link to the master branch, which is not ideal, as this might appear in other branches, and ultimately in a release version of GAP, where this makes even less sense. But I think this will just work, too: `[the reviewing process](#the-reviewing-process)`. Alternatively, just do without the link, and say:
> ... read the section "The reviewing process" later in this file ...